### DISCUSSION TITLE
How to setup an automation to control a light (brightness etc.) with one of the remotes?

### DISCUSSION THREAD

**Discussion by pvdw0310** [EPMatt/awesome-ha-blueprints#309] [Q&A] [answered]
### Blueprint name

Controller - IKEA E1524/E1810 TRÅDFRI Wireless 5-Button Remote

### Description

Thanks for the nice blueprint.
I use the ikea 5 button (up-down) to dim a zha zigbee dimmer. 

Is it possible to stop the down dimming at a minimum level and not till zero ( = light total out )? Because if the light is total dim out, it is difficult to put it on again. 

I'am new to use blueprint and don't know how to change it. 
Many thanks to help in advance.

### Additional Details

- [ ] I'd like to help developing this feature/improvement.

**Comment by EPMatt** [comment, marked as answer]
Hi @pvdw0310,

thank you so much for reaching out here and for your precious feedback! I'm sorry for my late response.
If you want to control a light or a group of lights with your controller, you can set up an automation with the [Light Hook](https://epmatt.github.io/awesome-ha-blueprints/docs/blueprints/hooks/light) blueprint and link it to the automation created with this blueprint. You can find more info in the Light Hook blueprint's docs.

Please let me know if you need any additional guidance to setup this blueprint. I'd be truly happy to help you. :)
Thanks!

**Reply by Zetto12** [reply]
Hey EPMatt,

I'm a bit confused how to link the light hook with the controller blueprint (Ikea E1744 in my case) do you have an example setup somewhere?

Thanks,
Martin
